### ISSUE TITLE
A11Y: WebView2 Sample is completely inaccessible with all the screen readers

### ISSUE THREAD

**Issue by akash07k** [microsoft/microsoft-ui-xaml#9263]
**Describe the bug**

The WebView2 example in both (XAML Controls Gallery and WinUI3 Controls Gallery) is completely inaccessible with all the screen readers.
**Steps to reproduce the bug**

1. Go to 'Start menu'
2. Search for "XAML Controls Gallery or WinUI3 Controls Gallery (Either of the app should be installed)
3. As soon as the app opens up, search for WebView2 and click the search result. Now navigate via tab inside the WebView2 Sample.
4. Observe with any Screen reader, nothing will be read out.
5. Now press arrow keys to read the web content.
6. Observe, the situation is still same.
7. Try to close the app via Alt+F4.
8. App won't close.
9. Please see the attached app too for demo.

**Expected behavior**

1. WebView2/Sample should be accessible and all the web content should be read by the screen reader.
2. We should be able to close the app/window via Alt+F4
**Actual behavior**

As soon as we tab inside the WebView2 control, our focus gets trapped, screen reader doesn't read anything via tab or arrow keys and we are unable to close the app with Alt+F4.
**Screenshots**

**Version Info**
- [ ] Xaml Controls Gallery - Downloaded from Store
    - If so, which version? (1.2.x -- Found on the Settings page of the app.)
- [x] WinUI 3 Controls Gallery - Downloaded from Store 
    - If so, which version? (1.3.x -- Found on the Settings page of the app.)
Microsoft.WinUI3ControlsGallery version 1.3.10.0

- [ ] App built locally from source 
  - If so, which branch? (WinUI 3 versions are built off the winui3 branch)

**Additional context**

Windows version:
- [x] Insider Build (22581)
- [ ] May 2021 Update (19043)
- [ ] October 2020 Update (19042)
- [ ] May 2020 Update (19041)
- [ ] November 2019 Update (18363)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

https://user-images.githubusercontent.com/6381747/161434117-343681bc-a6f4-4c79-a19d-0d443842ab71.mp4

[Scarlett.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/8404848/Scarlett.zip)

**Comment by akash07k** [comment]
No, not at all.
It is not related
> Hi I'm an AI powered bot that finds similar issues based off the issue title.
> 
> Please view the issues below to see if they solve your problem, and if the issue describes your problem please consider closing this one. Thank you!
> 
> ### Closed similar issues:
> * [Possible bug on webview2. (#5410)](https://github.com/microsoft/microsoft-ui-xaml/issues/5410),  similarity score: 0.73
> 
> > Note: You can give me feedback by thumbs upping or thumbs downing this comment.

**Comment by akash07k** [comment]
Any update @gregwoo-microsoft 
> Thanks for filing this bug @akash07k. We are currently looking into solving this issue. I'll close this issue when this is fixed.

**Comment by gregwoo-microsoft** [comment]
@akash07k thanks for following up. I'll defer to @karkarl who might have more insights on the progress made here.

**Comment by ranjeshj** [comment]
@akash07k does this still repro with latest WinAppSDK ? There are several fixes in this space over the past year. The Alt+f4 issue is a known issue.

**Comment by bkudiess** [comment]
Does not repro in current version, webview is accessible by screen readers. Alt+F4 not working is a core webview issue tracked outside of this repo (https://github.com/MicrosoftEdge/WebView2Feedback/issues/2961)
